src/seedu/addressbook/ui/TextUi.java

```diff
@@ -165,5 +166,18 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
+    
+    /**
+     * 
+     */
+    private String getPrintableString(Printable... printables) {
+    	StringBuilder builder = new StringBuilder();
+    	for (Printable printable: printables) {
+    		builder.append(printable.getPrintableString());
+    		builder.append(". ");
```

Comment:
I think this result in an extra " " at the end. Take a look @ StringJoiner or String.join that was mentioned in tutorial :)